deploy/examples/pool-device-health-metrics.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  # If the built-in Ceph pool for health metrics needs to be configured with alternate
+  # settings, create this pool with any of the pool properties. Create this pool immediately
+  # with the cluster CR, or else some properties may not be applied when Ceph creates the
+  # pool by default.
+  name: device-health-metrics
```

Comment:
For my own edification, what properties can't be changed after the pool is created?